src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialRestorePageObject.java

```diff
@@ -24,7 +24,7 @@ public SpecialRestorePageObject(WebDriver driver) {
     super(driver);
   }
 
-  public void verifyArticleName(String articleName) {
+  public void verifyArticleNameInWgPageName(String articleName) {
```

Comment:
from what I see article name here is not taken from wgPageName so why did you rename this method?